src/test.py

```diff
@@ -12,6 +12,10 @@
 progress = ProgressBar("[Tutkain] Running tests...")
 
 
+def test_results_settings_key():
```

Comment:
This could be just `RESULTS_SETTINGS_KEY = "tutkain_clojure_test_results"` instead of a function, I think.